lib/libzpool/util.c

```diff
@@ -76,6 +76,12 @@ show_vdev_stats(const char *desc, const char *ctype, nvlist_t *nv, int indent)
 	char rops[6], wops[6], rbytes[6], wbytes[6], rerr[6], werr[6], cerr[6];
 	char *prefix = "";
 
+	v0 = umem_zalloc(sizeof (*v0), UMEM_NOFAIL);
+	if (!v0) {
+		printf("%s: not enough memory\n", __func__);
+		return;
+	}
+
```

Comment:
UMEM_NOFAIL really does mean it can't fail and the error handling can be dropped.  You'll also want to free this with `umem_free()` which does just call `free()`.